modules/bitgo/test/v2/unit/wallets.ts

```diff
@@ -490,15 +528,64 @@ describe('V2 Wallets:', function () {
 
       const wallets = new Wallets(bitgo, tsol);
 
-      await wallets.generateWallet({
+      const params = {
         label: 'tss wallet',
         passphrase: 'tss password',
-        multisigType: 'tss',
+        multisigType: 'tss' as any,
```

Comment:
Can't we avoid this any type casting?, multisigType should consider tss as valid

Reply:
pretty sure `as const` will work here